Source/XUnitExtensions/Lit/DiffCommand.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace XUnitExtensions.Lit {
+  public class DiffCommand : ILitCommand {
```

Comment:
Could you add a small comment explaining what this class does? Maybe a link to `https://man7.org/linux/man-pages/man1/diff.1.html`

Reply:
Added the following comment:

```
  /// <summary>
  /// This class implements the equivalent of the Unix 'diff' command that lit tests rely on,
  /// because 'diff' does not exist on Windows.
  /// </summary>
  ```